core/src/botany_core/runner.py

```diff
@@ -61,13 +61,23 @@ def build_result(result_type, score, traceback=None, invalid_move=None):
     states = [None, None]
     winning_scores = [1, -1]
     losing_scores = [-1, 1]
-    board = game.new_board()
-    move_list = []
+    #Create a variable for cycle to allow rerun of odd number of moves
+    the_cycle = [0, 1]
```

Comment:
`the_cycle` doesn't tell me very much about what this variable is for.  Could you rename to `player_ixs`?

Reply:
@inglesp Yes indeed. Updated it.